### PR TITLE
Fixes #8813: Use rudder agent commands in system techniques

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/8813

### CODE REVIEW

File: techniques/system/common/1.0/cron_setup.st [hunk 1]
```diff
@@ -55,13 +55,18 @@ bundle agent setup_cronjob
...
+      "/var/spool/cron/crontabs/root"
+        edit_defaults => noempty_backup,
+        edit_line => delete_lines_matching("0,5,10,15,20,25,30,35,40,45,50,55 \* \* \* \* /usr/bin/test -x /opt/rudder/bin/check-rudder-agent \&\& /opt/rudder/bin/check-rudder-agent >/dev/null"),
+
       # Add Rudder entry
       "/var/spool/cron/crontabs/root"
         create        => "true",
         perms         => mog("600", "root", "cron"),
-        edit_line     => insert_lines("0,5,10,15,20,25,30,35,40,45,50,55 * * * * /usr/bin/test -x /opt/rudder/bin/check-rudder-agent \&\& /opt/rudder/bin/check-rudder-agent >/dev/null"),
+        edit_line     => insert_lines("0,5,10,15,20,25,30,35,40,45,50,55 * * * * /usr/bin/test -x /opt/rudder/bin/check-rudder-agent \&\& /opt/rudder/bin/rudder agent check >/dev/null"),
```
Comment (jooooooon) [inline techniques/system/common/1.0/cron_setup.st]: You don't need the -x test on /opt/rudder/bin/check-rudder-agent if you're not going to call it :)

File: techniques/system/common/1.0/cron_setup.st [hunk 2]
```diff
@@ -55,13 +55,18 @@ bundle agent setup_cronjob
         edit_defaults => noempty_backup,
         edit_line => delete_lines_matching("0,5,10,15,20,25,30,35,40,45,50,55 \* \* \* \* if \[ -x /opt/rudder/bin/check-rudder-agent \]; then /opt/rudder/bin/check-rudder-agent( >/dev/null)?; fi");
 
+      # Cleanup the pre-3.3 crontab
```
Comment (jooooooon) [inline techniques/system/common/1.0/cron_setup.st]: This is going to be 4.0, we might as well start calling it that

File: techniques/system/common/1.0/rudder_agent_community_cron.st [hunk 3]
```diff
@@ -9,4 +9,4 @@
 # /opt/rudder/etc/rudder-restart-message.txt file with your custom message inside. It will be sent by mail
```
Comment (jooooooon) [inline techniques/system/common/1.0/rudder_agent_community_cron.st]: You could change the line above (touch /opt/rudder/etc/disable-agent) to say "run rudder agent disable" while you're here

File: techniques/system/common/1.0/update.st [hunk 4]
```diff
@@ -464,7 +464,7 @@ bundle agent check_uuid
     wrong_uuid::
       "FATAL: The UUID in ${g.uuid_file} (${g.uuid}) doesn't match the UUID recorded for this node in your Rudder server (&UUID&)";
     no_uuid::
-      "FATAL: The UUID file ${g.uuid_file} is missing. Please run check-rudder-agent to generate a new one or restore it from a backup.";
+      "FATAL: The UUID file ${g.uuid_file} is missing. Please run rudder agent check to generate a new one or restore it from a backup.";
```
Comment (jooooooon) [inline techniques/system/common/1.0/update.st]: Can you add quotes around this, now it just looks like a succession of words? "Please run 'rudder agent check' to ..."

**Comment by peckpeck** [comment]
Commit modified

File: techniques/system/common/1.0/rudder_agent_community_cron.st [hunk 5]
```diff
@@ -2,11 +2,11 @@
 #
 # Will manually run cf-agent in case cf-execd is no longer running. cf-agent will fire up a new cf-execd.
 #
-# To temporarily avoid this behaviour, touch /opt/rudder/etc/disable-agent.
+# To temporarily avoid this behaviour, run 'ruddre agent disable'
```
Comment (jooooooon) [inline techniques/system/common/1.0/rudder_agent_community_cron.st]: s/ruddre/rudder/

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**